test/Spec/ExampleSpec.hs

```diff
@@ -0,0 +1,11 @@
+module Spec.ExampleSpec where
+
+import Test.Hspec
+
+spec :: Spec
+spec = describe "ExampleSpec" $ do
+  it "can parse integers" $ do
+    read "10" `shouldBe` (10 :: Int)
+
+  it "can parse floating-point numbers" $ do
+    read "2.5" `shouldBe` (2.5 :: Float)
```

Comment:
:information_source: Output of this test. Please double check if this is expected.

## Execution

```
cabal test
Resolving dependencies...
Build profile: -w ghc-9.2.4 -O1
In order, the following will be built (use -v for more details):
 - hspec-formatters-codewars-0.1.0 (test:hspec-formatters-codewars-spec) (first run)
Preprocessing test suite 'hspec-formatters-codewars-spec' for hspec-formatters-codewars-0.1.0..
Building test suite 'hspec-formatters-codewars-spec' for hspec-formatters-codewars-0.1.0..
Running 1 test suites...
Test suite hspec-formatters-codewars-spec: RUNNING...
Test suite hspec-formatters-codewars-spec: PASS
Test suite logged to:
/home/andreas/dev/repository/hspec-formatters-codewars/dist-newstyle/build/x86_64-linux/ghc-9.2.4/hspec-formatters-codewars-0.1.0/t/hspec-formatters-codewars-spec/test/hspec-formatters-codewars-0.1.0-hspec-formatters-codewars-spec.log
1 of 1 test suites (1 of 1 test cases) passed.
```

## Log output

```
Test suite hspec-formatters-codewars-spec: RUNNING...

<DESCRIBE::>Spec.Example

<DESCRIBE::>ExampleSpec

<IT::>can parse integers

<PASSED::>Test Passed

<COMPLETEDIN::>

<IT::>can parse floating-point numbers

<PASSED::>Test Passed

<COMPLETEDIN::>

<COMPLETEDIN::>

<COMPLETEDIN::>
Test suite hspec-formatters-codewars-spec: PASS
Test suite logged to: /home/andreas/dev/repository/hspec-formatters-codewars/dist-newstyle/build/x86_64-linux/ghc-9.2.4/hspec-formatters-codewars-0.1.0/t/hspec-formatters-codewars-spec/test/hspec-formatters-codewars-0.1.0-hspec-formatters-codewars-spec.log
```

Reply:
The output looks correct.

(We should add tests later.)